extensions/mini/__init__.py

```diff
@@ -97,3 +102,22 @@ def register(self):
             await message.channel.send(embed=embed)
         else:
             await message.channel.send("Role not found")
+
+    async def avatar(self, args, message):
+        parser = argparse.ArgumentParser(description=self.bot.processOutput("Shows an clickable avatar of a user", message))
+        parser.add_argument("avatar", nargs="?", default=str(message.author.id), help=self.bot.processOutput("The user of the avatar you want to get", message))
+        args = parser.parse_args(args)
+        avatar = message.channel.guild.get_member(self.bot.getIDFromTag(args.avatar))

```

Comment:
This variable should be `user` or `member`, not `avatar`. It could confuse someone who tries to update this function in the future.